ethers-ext/src/ethers/signer.ts

```diff
@@ -282,7 +282,7 @@ export class Wallet extends EthersWallet {
     const signedTx = await this.signTransactionAsFeePayer(tx);
 
     if (!KlaytnTxFactory.has(tx.type)) {
-      return await this.provider.sendTransaction(signedTx);
+      throw new Error(`Fee payer can not sign with Legacy transaction`);
```

Comment:
Isn't the error checked in signTransactionAsFeePayer (line 247)?

Reply:
That's a good comment.
Since signTransactionAsFeePayer is already called on line 282, an error occurs in signTransactionAsFeePayer before this error occurs.
I will delete 284~286 lines.
Thanks.